guides/_subsections/navigate-the-dashboard.md

```diff
@@ -0,0 +1,421 @@
+
+#### [Create a new Dashboard Layout](#create-a-new-dashboard-layout)
+<ol>
+    <li>
+        {% include_relative _subsections/common/layout-selector.md %}
+    </li>
+    <li>
+        {% include howto_step.html
+        summary="Click <i>New Layout</i>"
+        image-name="2.3.0/dashboard/dashboard-blanklayout.png"
+        text="Clicking <i>New Layout</i> will create a new blank layout and switch the dashboard view to the new layout. The layout should be called something like <code>Custom1</code>."
+        %}
+    </li>
+    <li>
+        {% include howto_step.html
+        summary="<i>(Optional)</i> Rename layouts"
+        image-name="2.3.0/dashboard/dashboard-renamelayout.png"
+        caption="Click the pencil button to rename the currently selected layout."
+        text="You are able to rename layouts by clicking the pencil button next to the layout selector. This will rename the currently selected layout. You can also rename layouts within the layout selector dropdown itself."
+        %}
+    </li>
+    <li>
+        {% include howto_step.html
+        summary="<i>(Optional)</i> Delete layouts"
+        image-name="2.3.0/dashboard/dashboard-deletelayout.png"
+        caption="Click 🗑️ <i>Delete</i> to delete the currently selected layout."
+        text="Deletion is similar to renaming. Click the trash can icon with the text <i>Delete</i> next to the layout selector to delete the currently selected layout. You can also delete layouts within the layout selector dropdown itself."
+        %}
+    </li>
+</ol>
+
+#### [Set a layout as a template](#set-a-layout-as-a-template)
+<ol>
+    <li>
+        {% include howto_step.html
+        summary="Save a layout as a template"
+        image-name="2.3.0/dashboard/dashboard-layout-setastemplate.png"
+        caption='Click more options "⋮" on the dashboard toolbar, then click <i>Set as template</i> to set the desired layout as a template.'
+        text="The layout template will be saved as a <i>User-submitted</i> template in the template picker."
+        %}
+    </li>
+</ol>
+
+#### [Create a new Dashboard Layout from a template](#create-a-new-dashboard-layout-from-a-template)
+
+<ol>
+    <li>
+        {% include_relative _subsections/common/layout-selector.md %}
+    </li>
+    <li>
+        {% include howto_step.html
+        summary="Select <i>Choose Template</i>"
+        image-name="2.3.0/dashboard/dashboard-layoutselector-options.png"
+        text="Click the expandable menu on <i>New Layout</i> button and select <i>Choose Template</i>. This will open the template picker."
+        %}
+    </li>
+    <li>
+        {% capture template-picker-guide-text %}
+        <p>
+            The template picker displays all the available templates. Templates are categorized into 3 groups.
+            <ol>
+                <li>
+                    <b>Suggested:</b> Templates that are suggested for you based on recent activity.
```

Comment:
Is this actually based on recent activity? I missed that part...

Reply:
Well technically it is based on your most recently used layouts, we just keep the last 5 recently used templates in localStorage, but it sounds cooler if we say "recent activity" and its technically somwhat true? No ML component at all towards this :-)